docker/web/development/Dockerfile

```diff
@@ -6,17 +6,20 @@ RUN apk add --no-cache --virtual .build-deps \
         icu-dev \
         libpq-dev \
         libzip-dev \
+        openldap-dev \
         sqlite-dev \
     && apk add --no-cache --virtual .runtime-programs \
         git \
         icu-data-full \
+        libldap \
```

Comment:
`libldap` is automatically included in `.gewisdb-runtime-deps`, so you do not have to explicitly here. Same applies to the production Dockerfile.